spec/components/post_creator_spec.rb

```diff
@@ -1141,6 +1141,26 @@
       post_creator = PostCreator.new(user, title: '', raw: '')
       expect { post_creator.create! }.to raise_error(ActiveRecord::RecordNotSaved)
     end
+
+    it "does not generate an alert for empty posts" do
+      SiteSetting.queue_jobs = false
```

Comment:
oh ... sorry ... but @eviltrout just killed this setting, can you migrate to the new pattern?